idb/postgres/postgres.go

```diff
@@ -1659,8 +1673,14 @@ func (db *IndexerDb) GetAccounts(ctx context.Context, opts idb.AccountQueryOptio
 	if err != nil {
 		err = fmt.Errorf("account round err %v", err)
 		out <- idb.AccountRow{Error: err}
+		// Because we return a channel into a "callWithTimeout" function,
```

Comment:
I would put this one line above in case the caller cancels the context right after receiving an error on the channel.